Configuration.md

```diff
@@ -329,27 +330,29 @@ In some cases, you may want to start your program with the external builder and
 
 - `java.debug.settings.forceBuildBeforeLaunch` - Force building the workspace before launching java program, defaults to `true`. Sometimes you may be bothered with the message *"Build failed, do you want to continue?"*, you could disable this setting to suppress the message.
 
+- `java.debug.settings.onBuildFailureProceed` - Sometimes you may be bothered with the message *"Build failed, do you want to continue?"*, you could use this setting to suppress the message and proceed.
+
 - `java.debug.settings.hotCodeReplace` - Reload the changed Java classes during debugging, defaults to `manual`. It supports `manual`, `auto`, `never`.
-  - `manual` - Click the toolbar to apply the changes.  
+  - `manual` - Click the toolbar to apply the changes.
   ![hcr](https://user-images.githubusercontent.com/14052197/67256313-f5697600-f4b8-11e9-9db6-54540b6350ad.png)
   - `auto` - Automatically apply the changes after saved.
   - `never` - Never apply the changes.
 
 - `java.debug.settings.enableRunDebugCodeLens` - Enable the code lens provider for the *Run* and *Debug* buttons over main entry points, defaults to `true`.
-  - `true` - Show the code lens.  
+  - `true` - Show the code lens.
   ![codelens](https://user-images.githubusercontent.com/14052197/67256585-83922c00-f4ba-11e9-883f-2b3de3db2dfa.png)
-  - `false` - Show the Run/Debug link in the hover.  

```

Comment:
Again, please keep the tailing 2 spaces, which is used for newline.

Reply:
All formatting adjusted to minimal changes.